src/pages/trader-tools/common/_style.ts

```diff
@@ -40,43 +40,38 @@ export const StyledHeader = styled(Header)`
 `
 
 export const BreadCrumbContainer = styled(Container)`
-    margin-top: 2.5rem;
-    width: 100%;
-    white-space: nowrap;
+    margin-top: 2.4rem;
 
-    @media (min-width: 1280px) and (max-width: 1600px) {
-        padding: 0;
-        margin-right: 27vw;
-    }
-
-    @media (min-width: 1600px) and (max-width: 1920px) {
-        padding: 0;
-        margin-right: 48vw;
+    @media ${device.laptopL} {
+        width: 100%;
     }
-
-    @media (min-width: 1920px) and (max-width: 2600px) {
-        padding: 0;
-        margin-right: 65vw;
+    @media ${device.laptopM} {
+        width: 90%;
     }
 `
 
 export const StyledSection = styled(SectionContainer)`
     position: relative;
-    padding: 8rem 5rem;
+    padding: 8rem 0;
 
-    @media ${device.desktopS} {
-        padding: 40px 0;
+    @media ${device.laptop} {
+        padding: 40px 8px;
+        width: 100%;
+    }
+
+    @media (min-width: 1024px) and (max-width: 1280px) {
+        padding: 40px 56px;
         width: 100%;
     }
 `
 
 export const SectionSubtitle = styled(Header)`
-    display: flex;
-    justify-content: center;
-    align-items: center;
-
-    @media (min-width: 1400px) {
-        padding: 0 20%;
+    width: 79.2rem;
+    margin: 0 auto;
+    line-height: 1.5;
+    @media ${device.tablet} {
```

Comment:
Can we add a space before `@media` here.  Also why do we need the `79.2rem` width?

Reply:
Yes will add a space, idk about the 79.2rem it was already like this in codebase. I had to start from the beginning so I changed back to the original settings most of the lines